source/cli.js

```diff
@@ -95,6 +96,7 @@ updateNotifier({pkg: cli.pkg}).notify();
 		process.exit(0);
 	}
 
+	console.log();
```

Comment:
This doesn't fix anything. I added it because currently it becomes hard (for me, at least) to read the task titles when they are all stuck together, and there's no spacing between them and the previous logs.

Reply:
It’s best to add a comment like “adds a line break for readability” or something